test/libsolidity/semanticTests/externalContracts/openzeppelin/Ownable.sol

```diff
@@ -0,0 +1,25 @@
+==== ExternalSource: _contracts/import.sol ====
+import "_contracts/access/Ownable.sol";
+
+// Make Box inherit from the Ownable contract
+contract Box is Ownable {
+    uint256 private value;
+
+    event ValueChanged(uint256 newValue);
+
+    // The onlyOwner modifier restricts who can call the store function
+    function store(uint256 newValue) public onlyOwner {
+        value = newValue;
+        emit ValueChanged(newValue);
+    }
+
+    function retrieve() public view returns (uint256) {
+        return value;
+    }
+}
+
+// ====
+// compileViaYul: also
+// ----
+// store(uint256): 123
+// retrieve() -> 123
```

Comment:
Maybe we should add an isoltest builtin that allows changing the current used account.